<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>349</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Form{
background-repeat: no-repeat;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#groupBox_5{
	border:none;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="BodyLabel" name="BodyLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>welcome to wisdom system</string>
        </property>
        <property name="pixelFontSize" stdset="0">
         <number>20</number>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="styleSheet">
        <string notr="true">#groupBox_4{
	border:none;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>-33</y>
          <width>261</width>
          <height>441</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#groupBox_3{
	border:none;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>0</y>
          <width>90</width>
          <height>32</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#groupBox_7{
	border:none;
}</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="PushButton" name="minimize_button">
           <property name="styleSheet">
            <string notr="true">#minimize_button{
	background-color:transparent;
	border:none;
}
#minimize_button：hover{
	padding-bottom:5px;

}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PushButton" name="maxmize_button">
           <property name="styleSheet">
            <string notr="true">#maxmize_button{
	background-color:transparent;
	border:none;
}
#maxmize_button：hover{
	padding-bottom:5px;

}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="PushButton" name="close_button">
           <property name="styleSheet">
            <string notr="true">#close_button{
	background-color:transparent;
	border:none;
}
#close_button：hover{
	padding-bottom:5px;

}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>5</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#groupBox_2{
		border:none;
		
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="PasswordLineEdit" name="PasswordLineEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>160</y>
          <width>158</width>
          <height>33</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>密码</string>
        </property>
       </widget>
       <widget class="LineEdit" name="UserEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>161</width>
          <height>33</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>账号</string>
        </property>
       </widget>
       <widget class="ImageLabel" name="user_photo_label">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>0</y>
          <width>100</width>
          <height>72</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>72</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#groupBox{
	border:none;
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="PushButton" name="LoginButton">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">PushButton, ToolButton, ToggleButton, ToggleToolButton {
    color: black;
    background: rgba(255, 255, 255, 0.7);
    border: 1px solid rgba(0, 0, 0, 0.073);
    border-bottom: 1px solid rgba(0, 0, 0, 0.183);
    border-radius: 5px;
    /* font: 14px 'Segoe UI', 'Microsoft YaHei'; */
    padding: 5px 12px 6px 12px;
    outline: none;
}


ToolButton {
    padding: 5px 9px 6px 8px;
}

PushButton[isPlaceholderText=true] {
    color: rgba(0, 0, 0, 0.6063);
}

PushButton[hasIcon=false] {
    padding: 5px 12px 6px 12px;
}

PushButton[hasIcon=true] {
    padding: 5px 12px 6px 36px;
}

DropDownToolButton, PrimaryDropDownToolButton {
    padding: 5px 31px 6px 8px;
}

DropDownPushButton[hasIcon=false],
PrimaryDropDownPushButton[hasIcon=false] {
    padding: 5px 31px 6px 12px;
}

DropDownPushButton[hasIcon=true],
PrimaryDropDownPushButton[hasIcon=true] {
    padding: 5px 31px 6px 36px;
}

PushButton:hover, ToolButton:hover, ToggleButton:hover, ToggleToolButton:hover {
    background: rgba(249, 249, 249, 0.5);
}

PushButton:pressed, ToolButton:pressed, ToggleButton:pressed, ToggleToolButton:pressed {
    color: rgba(0, 0, 0, 0.63);
    background: rgba(249, 249, 249, 0.3);
    border-bottom: 1px solid rgba(0, 0, 0, 0.073);
	padding-top:5px;
	padding-left:1px;

}

PushButton:disabled, ToolButton:disabled, ToggleButton:disabled, ToggleToolButton:disabled {
    color: rgba(0, 0, 0, 0.36);
    background: rgba(249, 249, 249, 0.3);
    border: 1px solid rgba(0, 0, 0, 0.06);
    border-bottom: 1px solid rgba(0, 0, 0, 0.06);
}


PrimaryPushButton,
PrimaryToolButton,
ToggleButton:checked,
ToggleToolButton:checked {
    color: white;
    background-color: #009faa;
    border: 1px solid #00a7b3;
    border-bottom: 1px solid #007780;
}

PrimaryPushButton:hover,
PrimaryToolButton:hover,
ToggleButton:checked:hover,
ToggleToolButton:checked:hover {
    background-color: #00a7b3;
    border: 1px solid #2daab3;
    border-bottom: 1px solid #007780;
}

PrimaryPushButton:pressed,
PrimaryToolButton:pressed,
ToggleButton:checked:pressed,
ToggleToolButton:checked:pressed {
    color: rgba(255, 255, 255, 0.63);
    background-color: #3eabb3;
    border: 1px solid #3eabb3;
}

PrimaryPushButton:disabled,
PrimaryToolButton:disabled,
ToggleButton:checked:disabled,
ToggleToolButton:checked:disabled {
    color: rgba(255, 255, 255, 0.9);
    background-color: rgb(205, 205, 205);
    border: 1px solid rgb(205, 205, 205);
}

SplitDropButton,
PrimarySplitDropButton {
    border-left: none;
    border-top-left-radius: 0;
    border-bottom-left-radius: 0;
}

#splitPushButton,
#splitToolButton,
#primarySplitPushButton,
#primarySplitToolButton {
    border-top-right-radius: 0;
    border-bottom-right-radius: 0;
}

#splitPushButton:pressed,
#splitToolButton:pressed,
SplitDropButton:pressed {
    border-bottom: 1px solid rgba(0, 0, 0, 0.183);
}

PrimarySplitDropButton:pressed {
    border-bottom: 1px solid #007780;
}

#primarySplitPushButton, #primarySplitToolButton {
    border-right: 1px solid #3eabb3;
}

#primarySplitPushButton:pressed, #primarySplitToolButton:pressed {
    border-bottom: 1px solid #007780;
}

HyperlinkButton {
    /* font: 14px 'Segoe UI', 'Microsoft YaHei'; */
    padding: 6px 12px 6px 12px;
    color: #009faa;
    border: none;
    border-radius: 6px;
    background-color: transparent;
}

HyperlinkButton[hasIcon=false] {
    padding: 6px 12px 6px 12px;
}

HyperlinkButton[hasIcon=true] {
    padding: 6px 12px 6px 36px;
}

HyperlinkButton:hover {
    color: #009faa;
    background-color: rgba(0, 0, 0, 10);
    border: none;
}

HyperlinkButton:pressed {
    color: #009faa;
    background-color: rgba(0, 0, 0, 6);
    border: none;
}

HyperlinkButton:disabled {
    color: rgba(0, 0, 0, 0.43);
    background-color: transparent;
    border: none;
}


RadioButton {
    min-height: 24px;
    max-height: 24px;
    background-color: transparent;
    font: 14px 'Segoe UI', 'Microsoft YaHei', 'PingFang SC';
    color: black;
}

RadioButton::indicator {
    width: 18px;
    height: 18px;
    border-radius: 11px;
    border: 2px solid #999999;
    background-color: rgba(0, 0, 0, 5);
    margin-right: 4px;
}

RadioButton::indicator:hover {
    background-color: rgba(0, 0, 0, 0);
}

RadioButton::indicator:pressed {
    border: 2px solid #bbbbbb;
    background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5,
            stop:0 rgb(255, 255, 255),
            stop:0.5 rgb(255, 255, 255),
            stop:0.6 rgb(225, 224, 223),
            stop:1 rgb(225, 224, 223));
}

RadioButton::indicator:checked {
    height: 22px;
    width: 22px;
    border: none;
    border-radius: 11px;
    background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5,
            stop:0 rgb(255, 255, 255),
            stop:0.5 rgb(255, 255, 255),
            stop:0.6 #009faa,
            stop:1 #009faa);
}

RadioButton::indicator:checked:hover {
    background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5,
            stop:0 rgb(255, 255, 255),
            stop:0.6 rgb(255, 255, 255),
            stop:0.7 #009faa,
            stop:1 #009faa);
}

RadioButton::indicator:checked:pressed {
    background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5,
            stop:0 rgb(255, 255, 255),
            stop:0.5 rgb(255, 255, 255),
            stop:0.6 #009faa,
            stop:1 #009faa);
}

RadioButton:disabled {
    color: rgba(0, 0, 0, 110);
}

RadioButton::indicator:disabled {
    border: 2px solid #bbbbbb;
    background-color: transparent;
}

RadioButton::indicator:disabled:checked {
    border: none;
    background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5,
            stop:0 rgb(255, 255, 255),
            stop:0.5 rgb(255, 255, 255),
            stop:0.6 rgba(0, 0, 0, 0.2169),
            stop:1 rgba(0, 0, 0, 0.2169));
}

TransparentToolButton,
TransparentToggleToolButton,
TransparentDropDownToolButton,
TransparentPushButton,
TransparentDropDownPushButton,
TransparentTogglePushButton {
    background-color: transparent;
    border: none;
    border-radius: 5px;
    margin: 0;
}

TransparentToolButton:hover,
TransparentToggleToolButton:hover,
TransparentDropDownToolButton:hover,
TransparentPushButton:hover,
TransparentDropDownPushButton:hover,
TransparentTogglePushButton:hover {
    background-color: rgba(0, 0, 0, 9);
    border: none;
}

TransparentToolButton:pressed,
TransparentToggleToolButton:pressed,
TransparentDropDownToolButton:pressed,
TransparentPushButton:pressed,
TransparentDropDownPushButton:pressed,
TransparentTogglePushButton:pressed {
    background-color: rgba(0, 0, 0, 6);
    border: none;
}

TransparentToolButton:disabled,
TransparentToggleToolButton:disabled,
TransparentDropDownToolButton:disabled,
TransparentPushButton:disabled,
TransparentDropDownPushButton:disabled,
TransparentTogglePushButton:disabled {
    background-color: transparent;
    border: none;
}


PillPushButton,
PillPushButton:hover,
PillPushButton:pressed,
PillPushButton:disabled,
PillPushButton:checked,
PillPushButton:checked:hover,
PillPushButton:checked:pressed,
PillPushButton:disabled:checked,
PillToolButton,
PillToolButton:hover,
PillToolButton:pressed,
PillToolButton:disabled,
PillToolButton:checked,
PillToolButton:checked:hover,
PillToolButton:checked:pressed,
PillToolButton:disabled:checked {
    background-color: transparent;
    border: none;
}
</string>
          </property>
          <property name="text">
           <string>登录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PushButton" name="EnrollButton">
          <property name="sizeIncrement">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>注册</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PasswordLineEdit</class>
   <extends>LineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
